R2API/ItemAPI.cs

```diff
@@ -183,6 +187,29 @@ public static class ItemAPI {
             return false;
         }
 
+        /// <summary>
+        /// Add a custom item tag to the list of item tags.
+        /// If this is called after the ItemCatalog inits then this will return -1 and ignore the custom item tag.
+        /// </summary>
+        /// <param name="name">The tag to add.</param>
+        /// <returns>ItemTag value if added or already existent, (-1) cast to ItemTag otherwise</returns>
+        public static ItemTag AddItemTag(string name){
+            if (!Loaded) {
+                throw new InvalidOperationException($"{nameof(ItemAPI)} is not loaded. Please use [{nameof(R2APISubmoduleDependency)}(nameof({nameof(ItemAPI)})]");
+            }
+
+            if (!CatalogBlockers.GetAvailability<ItemDef>()) {
+                R2API.Logger.LogError($"Too late ! Tried to add itemTag: {name} after the ItemCatalog has Initialized!");
```

Comment:
No space before '!'

Reply:
It's consistent with the rest of the file,though I don't know if that is reason enough to keep it.

Reply:
pre commit dotnet format should be there since a long time, its fine if code style is not fully respected, it'll get cleaned up eventually by someone tool